src/services/httpsRequestUtils.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { base64Encode } from './httpsRequestUtils'
+
+describe('#base64Encode', () => {
+  it('returns the correctly encoded value along with the "Basic " prefix', () => {
+    const value = 'helloWorld'
```

Comment:
Might be better if you used something closer to what you'll actually be using in practice, which I think is a token and username or something? Obviously not a real token, but just something that has the same shape if you can?

Reply:
done!